venues/ACM.org/SIGIR/Badging/process/decisionNotification.js

```diff
@@ -0,0 +1,31 @@
+function() {
+  var or3client = lib.or3client;
+  console.log('Tag process');
+
+  var SHORT_PHRASE = 'ACM SIGIR Badging';
+  
+  var tagsP = or3client.or3request(or3client.tagsUrl + '?forum=' + note.forum, {}, 'GET', token)
+  .then(result => result.tags.map(tagItem => tagItem.tag));
+
+  var forumNotesP = or3client.or3request(or3client.notesUrl + '?id=' + note.forum, {}, 'GET', token);
+
+  Promise.all([
+    forumNotesP,
+    tagsP
+  ])
+  .then(result => {
+    var forumNote = result[0].notes[0];
+    var assignedBadges = result[1];
+    var authorMail;
+    var authorMail = {
+      groups: forumNote.content.authorids,
+      subject: '[' + SHORT_PHRASE + '] A new badge has been posted on your submission. Title : "' + forumNote.content.title + '"',
+      message: 'Your submission to ' + SHORT_PHRASE + ' has received a new badge.\n\nSubmission title: ' + forumNote.content.title + 
+      '\n\nRequested Badges: ' + forumNote.content["requested badges"] + '\n\nAssigned Badges: ' + assignedBadges
```

Comment:
What is the type of assignedBadges and content['requested badges']? If they are objects or arrays then they will just appear as [Object object] in the email.

Reply:
Both forumNote.content["requested badges"] and assignedBadges are arrays of strings. This is how the email shows up with this:
`{"from":"openreview@local.openreview.net",
"fromname":"OpenReview",
"to":,author-email,
"subject":"[ACM SIGIR Badging] A new badge has been posted on your submission. Title : \"<paper title>\"",
"text":"Your submission to ACM SIGIR Badging has received a new badge.\n\nSubmission title: <paper title>\n\nRequested Badges: Artifacts Available,Results Replicated\n\nAssigned Badges: Results Replicated,Artifacts Available,Artifacts Evaluated – Functional and Reusable"}`

Reply:
I think it would be better if you called .join(', ') on those vars. The code would be more clear and the string would have the proper spaces.